code/modules/projectiles/guns/energy.dm

```diff
@@ -237,7 +237,7 @@
 	return
 
 /obj/item/weapon/gun/energy/lasgun/lasrifle
-	name = "\improper PP-73 lasrifle MK2"
+	name = "\improper TX-73 lasrifle MK2"
 	desc = "A multifunctional laser based rifle with an integrated mode selector. Ideal for any situation. Uses power cells instead of ballistic magazines."
```

Comment:
Tx does not follow the naming scheme please use a proper prefix

Reply:
I don't know how it'd fit into the naming scheme and I don't think it'd really matter since it isn't an available marine weapon, but sure.